test/cmd/templates.sh

```diff
@@ -11,11 +11,13 @@ os::log::install_errexit
 # This test validates template commands
 
 oc get templates
-oc create -f examples/sample-app/application-template-dockerbuild.json
+oc create -f examples/sample-app/application-template-dockerbuild.json 
+oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift
```

Comment:
@deads2k set it to start consistently, because some scripts were already monkeying with users and breaking assumptions for the others. I'd be fine with all subscripts starting as a normal user and escalating if they need to, but that was a bigger change